src/openapi/streaming/connection/transport/signalr-core-transport.js

```diff
@@ -0,0 +1,398 @@
+import fetch from '../../../../utils/fetch';
+import log from '../../../../log';
+import * as streamingTransports from '../../streamingTransports';
+import * as constants from '../constants';
+
+const LOG_AREA = 'SignalrCoreTransport';
+const NOOP = () => {};
+const RECONNECT_DELAYS = [2000, 3000, 5000, 10000, 20000];
+
+/**
+ * Handles any signal-r log, and pipes it through our logging.
+ * @param message
+ */
+function handleLog(level, message) {
+    if (level < window.signalrCore.LogLevel.Error) {

```

Comment:
you were going to change this to warnings.

Reply:
Done